docs/generate.py

```diff
@@ -0,0 +1,193 @@
+import inspect
+import pkgutil
+from importlib import import_module
+from pathlib import Path
+from types import ModuleType
+from typing import List
+
+import mkdocs_gen_files
+
+import streamlit_extras
+
+extra_modules_names = [
+    extra.name
+    for extra in pkgutil.iter_modules(streamlit_extras.__path__)
+    if extra.name != "query_string"
+]
```

Comment:
Ah. I excluded that one for some dependency issue - adding a todo that this should be explored more. 
```suggestion
# TODO: Check why query_string fails
extra_modules_names = [
    extra.name
    for extra in pkgutil.iter_modules(streamlit_extras.__path__)
    if extra.name != "query_string"
]
```